chapters/data-collection.tex

```diff
@@ -491,187 +493,241 @@ \section{Handling data securely}
+If your data collection service allows you to browse data in the browser, 
+then the encryption is only implemented correctly if you are asked for the key each time.
+
+Finally, the data security standards that apply when receiving confidential data from the field
+also apply when transferring confidential data to the field.
+In some survey software, 
+you can use the same encryption that allows you to receive data securely
+from the field, to also send confidential data,
+such as an identifying list of respondents, to the field.
+
+
+\subsection{Receiving or sharing data securely}
+
+If you receive confidential data from a partner organization 
+or share confidential data with your research team over the internet,
+it must always be encrypted.  
+The best practice is to reduce the amount of confidential data
+needed to be shared or transfer.
+If you do not need the confidential information you will simplify your workflow
+by collecting or receiving data without any confidential information in the first place.
+This should always be your first option when possible.
+
+Usually we need to deal with confidential data at some point in the project, 
+and then the second best practice is create two versions of your data. 
+One version of your data will still have the confidential information,
+and it will need to be encrypted at all times. 
+This version should only be used when the confidential information is actually needed.
+In the second version of your data all confidential data should be removed.
+This version does not have to be encrypted 
+and should be what you use on a day-to-day basis.
+
+The most common reason a data is confidential is that it includes personal identifiers. 
+Removing identifying information is called de-identification,
+and we will discuss how to do that in the next chapter. 
+De-identifying the data, at the earliest possible opportunity 
+will simplify the workflow without compromising data security,
+as de-identified data no longer must be encrypted.
+Another method of making data less confidential is to aggregate the data, 
+meaning that sums and averages of group of observations are calculated
+and only those numbers are reported.
+Any of these techniques can be used either before the data is shared with the research team,
+or when creating the second non-confidential version of the data set to be used in the analysis.
+If confidential information is directly required for the analysis itself,
+it will be necessary to keep at least a subset of the data encrypted through the data analysis process.
+
+There are multiple options on how to share data encrypted. 
+If you need to collaborate on a confidential data set 
+then you can create an encrypted folders using VeraCrypt as described above.
+An encrypted folder can be shared the same way you share a normal file
+and it can even be shared using insecure modes of communication
+such as email or third-party syncing services.
```

Comment:
Not clear to me how you would normally share files outside of email / sync software; suggest simplifying the wording as follows: 
An encrypted folder can safely be shared using insecure modes of communication such as email or third-party syncing services.

Reply:
implemented as suggested in e79e29f9ac376bfea501f95976ce45aaafddf1e6